hoodie-cli/src/main/java/com/uber/hoodie/cli/commands/RepairsCommand.java

```diff
@@ -57,8 +57,14 @@ public String deduplicate(@CliOption(key = {
           "sparkProperties"}, help = "Spark Properites File Path", mandatory = true) final String sparkPropertiesPath)
       throws Exception {
     SparkLauncher sparkLauncher = SparkUtil.initLauncher(sparkPropertiesPath);
-    sparkLauncher.addAppArgs(SparkMain.SparkCommand.DEDUPLICATE.toString(), duplicatedPartitionPath, repairedOutputPath,
-        HoodieCLI.tableMetadata.getBasePath());
+
+    SparkMain.HoodieDeduplicatePartitionCommandConfig config = new SparkMain.HoodieDeduplicatePartitionCommandConfig();
```

Comment:
Can you also move this config HoodieDeduplicatePartitionCommandConfig to DedupeSparkJob and if possible, move DedupeSparkJob to hoodie-utilities from cli